components/admin_console/password_settings.jsx

```diff
@@ -76,7 +76,7 @@ export default class PasswordSettings extends AdminSettings {
         config.PasswordSettings.Number = this.state.passwordNumber;
         config.PasswordSettings.Symbol = this.state.passwordSymbol;
 
-        config.ServiceSettings.MaximumLoginAttempts = this.parseIntNonZero(this.state.maximumLoginAttempts);
+        config.ServiceSettings.MaximumLoginAttempts = this.parseIntNonZero(this.state.maximumLoginAttempts, Constants.MAXIMUM_LOGIN_ATTEMPTS_DEFAULT);
```

Comment:
One of the tests wanted you to put in `ten` into the maximum login attempts field and save and ensure it gets reset to the default value. The default value on the server side is 10 however it kept resetting to 1 and I realized it's because we were not passing a default value into this function. Otherwise, the test kept failing.